administrator/components/com_menus/src/Model/MenutypesModel.php

```diff
@@ -496,7 +496,7 @@ protected function getTypeOptionsFromLayouts($component, $view)
                     $templateLayoutName = basename($layout, '.xml');
 
                     // Add to the list only if it is not a standard layout
-                    if (array_search($templateLayoutName, $layoutNames) === false) {
+                    if (!in_array($templateLayoutName, $layoutNames)) {
```

Comment:
```suggestion
                    if (!\in_array($templateLayoutName, $layoutNames)) {
```
Fix PHPCS failing in Drone.

Reply:
Thanks, it's resolved.